test/Microsoft.AspNetCore.Razor.Language.Test/TagHelperParseTreeRewriterTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Language.Test
+{
+    public class TagHelperParseTreeRewriterTests
```

Comment:
Add a few more test cases:

- Razor comments since you added support for that.
- Your parser feature flag being turned on/off

Reply:
Done